pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -181,9 +180,27 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(node *
 
 	util.SortVirtualHosts(virtualHosts)
 
-	if features.EnableFallthroughRoute.Get() && !useSniffing {
```

Comment:
So are we removing this configuration completely and enabling fall through routes by default - set to passthrough if sidecar's egress proxy is not present and to the destination provided by egress proxy if present. If yes, can we cleanup the flag as well?

Reply:
before, we had env var check and util.IsAllowAnyOutbound. So, even if env var was enabled, I dont think it would have worked unless the user explicitly set allow any right?

Reply:
That is correct. So we had two knobs  i.e. SidecarScope OutboundTrafficPolicy's mode - ALLOW_ANY would trigger this logic only if `EnableFallthroughRoute` is set to true (which is default). So user had choice at Pilot to turn that ALLOW_ANY feature off by setting `EnableFallthroughRoute` to `false` - Now it seems we are removing that control, which is fine, but let us just cleanup the flag as well?  

As a general question, do we give any guarantees around these flags or can be changed in newer releases and just note them in release notes?

Reply:
I think the flags are strictly experimental. I never understood why we needed that flag when we had an API level way of enabling it. As for cleaning up that flag, yep!

Reply:
Yes we can remove at any time, its just a fallback in case we introduce some new risky feature like this one. This flag was very very needed - there were multiple critical bugs associated with this feature, so I am glad we had a way to disable it, but now we can remove it